Microgame/IcePath/add-maps

Alexyzyz opened the pull request:
[IcePath] Add maps

Alexyzyz committed:
[IcePath] Add maps

-Reached the 8 maps per level quota.
-Pending maps review.

Gman8r reviewed (changes requested):
Get rid of the trailing .meta (or empty folder) for maps/0WIP

But before you do that tbh do some stage testing to take care of other fixes we may need: https://github.com/NitorInc/NitoriWare/wiki/ReimuDodge-Tutorial-Part-6#3-optional-microgame-practice

Alexyzyz committed:
[IcePath] Changes 1

-Got rid of the empty folder "Maps/0WIP".
-Adjusted some maps for improvement.

* Tested the game in MicrogamePractice.
* No issues regarding pausing, speed scaling, etc.
* Difficulty seems fairly balanced; I can consistently get to 50 before it becomes quite a challenge.
